designer/server/src/main/scala/pl/touk/nussknacker/ui/statistics/ScenarioStatistics.scala

```diff
@@ -213,3 +258,102 @@ case object ActiveScenarioCount    extends StatisticKey("s_a")
 case object NuSource               extends StatisticKey("source") // f.e docker, helmchart, docker-quickstart, binaries
 case object NuFingerprint          extends StatisticKey("fingerprint")
 case object NuVersion              extends StatisticKey("version")
+
+sealed abstract class ComponentKeys(val value: String, val name: String) extends StringEnumEntry {
+  override def toString: String = value
+}
+
+object ComponentKeys extends StringEnum[ComponentKeys] {
```

Comment:
IMHO we was close to best solution at the beginning because map was good but we should only do checking of custom component based on packageName, but only what we should additional was extract components names to components-api.

```scala
val componentsMappings = Map(
  ComponentsApi.ComponentEnum.AggregateName -> "some short name"
)
```
and then in test we should if all values from enum has mapping.

But we what should do now? Imho I would leave this enum but maybe we should take shortName like that: 
```scala
and then:
```scala
  // TODO extract components name to components-api
  private def mapNameToStat(name: String): Option[String] = {
    ComponentKeys.withNameOption(name).map(_.shortName)
  // shortNames can be invented by us
  }
```

Reply:
> extract components names to components-ap

We cannot do this, so imho what we should do is leave mapping like it was with vowels and check mapping in some integration test and remove this enum with mappings. Writing such test would be hard so I would leave todo and write such test later